### PR TITLE
Setup gh actions

### PR DESCRIPTION
**Pull request by thisismydesign**

### CODE REVIEW

**Comment by gomorizsolt** [comment]
Found the solution, see https://github.community/t5/GitHub-Actions/Github-Actions-on-push-not-master/m-p/29598#M13. String data types must be enclosed with quotes.

https://help.github.com/en/articles/contexts-and-expression-syntax-for-github-actions#literals

> You must use single quotes. Escape literal single-quotes with a single quote.

Therefore I've reverted the changes described above.

**Comment by gomorizsolt** [comment]
The following tasks are still left to be completed:
- Remove the whole `scripts/` folder as it'll be substituted with our custom action altogether.
 - https://github.com/c-hive/team-contribution-calendar/issues/53#issuecomment-541047774
- Question in https://github.com/c-hive/actions/pull/1 about how the actions should be organized.

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+    - name: Checkout
+      uses: actions/checkout@v1
+    - name: Build & distify
+      run: |
+        yarn install
+        yarn run distify
+    - name: Commit & push dist to master
+      uses: stefanzweifel/git-auto-commit-action@v2.2.0
+      with:
+        commit_message: 'Update distified version'
```
Comment (thisismydesign) [inline .github/workflows/ci.yml]: Should still make sure this doesn't trigger another run, otherwise, we end up in an infinite loop of commits.

Comment (gomorizsolt) [reply .github/workflows/ci.yml]: As per [`git-auto-commit-action`](https://github.com/stefanzweifel/git-auto-commit-action)'s docs, the action doesn't trigger another workflow run.

> The resulting commit will not trigger another GitHub Actions Workflow run!

Comment (thisismydesign) [reply .github/workflows/ci.yml]: Ah, cool

**Comment by thisismydesign** [comment]
Can't approve my own request but consider it approved :)

**Pull request merged**